src/sync/client.rs

```diff
@@ -98,20 +98,26 @@ impl Client {
             trace!("Sender quit");
         });
 
-        //Recver
+        //Reciver
         let receiver_connection = connection;
-        let receiver_client = client.clone();
+        //Clone as weak Arc , it will not add the count of client
```

Comment:
Could you add a detail comment here to describe why we should use weak Arc here?

Reply:
Ok  ,I have changed some comment